tools/eventbrite.md

```diff
@@ -8,13 +8,27 @@ TTS uses [Eventbrite](https://www.eventbrite.com/) to manage registrations for e
 
 ## Rules
 
-The following are required for accessibility reasons:
+The following are required for accessibility and other compliance reasons:
 
 - Include the following in the [event description](https://www.eventbrite.com/support/articles/en_US/Multi_Group_How_To/how-to-create-an-event?lg=en_US#2-2):
 
-  > Thank you for your interest in this event. Please contact us directly at [email] if you need accessible accommodations to be able to attend or assistance with registering for the event.
+  > Thank you for your interest in this event. Please contact us directly at [email] if you need accessible accommodations to register or be able to attend the event.
 
 - [Set the time limit to complete an order](https://www.eventbrite.com/support/articles/en_US/How_To/how-to-increase-or-decrease-the-amount-of-time-to-complete-an-order) to 20 minutes
+- [Include the following in the order form](https://www.eventbrite.com/support/articles/en_US/How_To/how-to-create-custom-questions-for-attendees):
+  - A notification of recording
+  
+    > By registering for this event, I acknowledge that this event may be recorded and that my image, voice, or name may be included in the recording or in any photographs of the event.
+
+  - Accessible accommodation options
+
+    > We provide a link to live captioning for all of our events. We also can offer direct assistance with registration, if needed. If you require additional accommodations to participate in this event, please contact us at [email] at least 3 business days prior to the event and let us know how we can best support your needs.
+    
+  - A Code of Conduct agreement
+
+    > By registering for this event, I abide by the following TTS Code of Conduct when participating in our webinars: https://handbook.tts.gsa.gov/code-of-conduct/.
+
+  - If attendees need to be federal employees, a question about whether they are federal or not
```

Comment:
This isn't a sufficient way to ensure that people are federal employees. If we are serious about this, we need to actually limit registrations/joining through MAX Authentication. Much larger problem, so we can leave this for now.